app/api/serializers.py

```diff
@@ -162,3 +162,28 @@ class Meta:
         model = Seq2seqAnnotation
         fields = ('id', 'text', 'user', 'document', 'prob')
         read_only_fields = ('user',)
+
+
+class RoleSerializer(serializers.ModelSerializer):
```

Comment:
We should add fields to the `UserSerializer` for each of the roles, like `is_project_admin (True, False)`, `is_label_approver (True, False)`, `is_annotator (True, False)` so that then we can replace the checks on `is_superuser` in the frontend with the more specific permissions check.

Reply:
Done in 9c85af4.